src/3p.js

```diff
@@ -104,7 +104,18 @@ function executeAfterWriteScript(win, fn) {
  * @param {string} src
  */
 export function validateSrcPrefix(prefix, src) {
-  if (src.indexOf(prefix) != 0) {
+  if (src.indexOf(prefix) !== 0) {
+    throw new Error('Invalid src ' + src);
+  }
+}
+
+/**
+ * Throws if the given src doesn't contain the string
+ * @param {string} string
+ * @param {string} src
+ */
+export function validateSrc(string, src) {
+  if (src.indexOf(string) === -1) {
```

Comment:
Woah, this file has no tests :( (Totally my fault)

Would you mind creating test-3p.js and adding a few?